wetterdienst/file_path_handling/path_handling.py

```diff
@@ -121,7 +130,32 @@ def build_local_filepath_for_station_data(folder: Union[str, Path]) -> Union[str
         a Path build upon the folder
     """
     local_filepath = Path(
-        folder, DWD_FOLDER_STATION_DATA, f"{DWD_FILE_STATION_DATA}{H5_FORMAT}"
+        folder, DWD_FOLDER_STATION_DATA, f"{DWD_FILE_STATION_DATA}{DataFormat.H5.value}"
+    ).absolute()
+
+    return local_filepath
+
+
+def build_local_filepath_for_radolan(
+        datetime: dt.datetime,
+        folder: Union[str, Path],
```

Comment:
I am very happy that you use Path here. But above yu are using PosixPath.

Reply:
Exactly because above I'm refering to a remote link path where we need forward slashes, while here for the local path we need backward slashes.

Reply:
Okay understand.